crescent/client.py

```diff
@@ -41,33 +32,29 @@
     INCLUDABLE = TypeVar("INCLUDABLE", bound=Includable[Any])
 
 
-__all___: Sequence[str] = ("Bot", "Mixin")
+__all___: Sequence[str] = ("Client", "CrescentAware")
 
 
-class Mixin(RESTAware, EventManagerAware):
+class CrescentAware(EventManagerAware, RESTAware, Protocol):
```

Comment:
"CrescentAware" feels inaccurate - this should mean that the class has the crescent object on it.

Reply:
What about `GatewayTraits`?